packages/features/borrow/src/borrow-fixed/hooks/use-borrow-fixed-chart.tsx

```diff
@@ -0,0 +1,63 @@
+import { useContext } from 'react';
+import { useTheme } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { ChartToolTipDataProps } from '@notional-finance/mui';
+import { BorrowFixedContext } from '../../borrow-fixed/borrow-fixed';
+import { useFCashMarket } from '@notional-finance/notionable-hooks';
+import {
+  getDateString,
+  formatNumberAsPercent,
+} from '@notional-finance/helpers';
+import { useMaturitySelect } from '@notional-finance/trade';
+
+export const useBorrowFixedChart = () => {
+  const theme = useTheme();
+  const {
+    state: { deposit },
+  } = useContext(BorrowFixedContext);
+  const { onSelect } = useMaturitySelect('Debt', BorrowFixedContext);
+  const fCashMarket = useFCashMarket(deposit?.currencyId);
+  let areaChartData: any[] = [];
+
+  if (fCashMarket) {
+    const {
+      poolParams: { perMarketfCash },
+    } = fCashMarket;
+
+    areaChartData = perMarketfCash.map((data) => {
+      const interestRate = fCashMarket.getSpotInterestRate(data.token);
+      return {
+        timestamp: data.maturity,
+        area: interestRate,
```

Comment:
Does this need to be named `area` or can we name it `interestRate` so it's more descriptive?

Reply:
It must be area.